wpilibc/src/main/native/include/frc/shuffleboard/SimpleWidget.h

```diff
@@ -27,17 +27,19 @@ class SimpleWidget final : public ShuffleboardWidget<SimpleWidget> {
 
   /**
    * Gets the NetworkTable entry that contains the data for this widget.
-   * The widget owns the entry.
+   * The widget owns the entry; the returned pointer's lifetime is the same as
```

Comment:
Can this be rephrased to be more clear? Is the user expected to maintain the lifetime of the widget object? An explanation of "X is the same as Y" doesn't help much when Y is also unknown.

Reply:
None of the C++ shuffleboard docs right now talk about lifetime, as they’re all copied from Java, where that’s not something that has to be worried about.  We should fix this more broadly, but it’s also outside the scope of this PR.